pkg/reconciler/cache/replication/replication_reconcile.go

```diff
@@ -106,7 +117,7 @@ func (c *controller) reconcile(ctx context.Context, gvrKey string) error {
 				if err != nil {
 					return nil, true, err
```

Comment:
why is this `return nil, true, err` and not `return nil, false, err` ?

Reply:
does not really matter. But yes, false would be more canonical

Reply:
Let's fix to be consistent?

Reply:
fixed